.github/CODEOWNERS

```diff
@@ -8,3 +8,14 @@
 
 # GitHub actions/checks approval
 /.github/ @aws-amplify/amplify-backend-admins
+
+# Packages used by console team.
+/packages/backend-secret @aws-amplify/amplify-backend @aws-amplify/amplify-studio-uibuilder
+/packages/backend-secret/API.md @aws-amplify/amplify-backend-api-approvers
```

Comment:
I left `API.md` intentionally like that.
This is to make sure that backend api approvers are mandatory.
API.md change always has code change that accompanies it, therefore console team is notified in that case as well.